Sources/XMLCoder/Auxiliaries/XMLChoiceCodingKey.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  XMLChoiceCodingKey.swift
+//  XMLCoder
+//
+//  Created by Benjamin Wetherfield on 7/17/19.
+//
+
+/// An empty marker protocol that can be used in place of `CodingKey`. It must be used when conforming a union-type–like enum with
```

Comment:
comment line too long, can this be split? Also, `XMLChoiceCodingKey` is only introduced to resolve the ambiguities with the unkeyed container? Can that be explained at least superficially in this doc comment here?

Reply:
Ah yes, of course. The new Xcode defaults to 10pt Helvetica for doc comments which crams a bunch more text in a given space. It's a bit unsettling. I'll revert that to get back my sense of reality and cut this down.

Reply:
And re: more helpful doc comment: 👍 .

Reply:
thanks! the most helpful setting I found for this is "Page guide at column", which doesn't even require changing the font size probably

Reply:
I've fleshed out the doc comment in 4a99e95. I'm not a documentation expert, so any words of wisdom / critique are more than welcome.

Reply:
And yes, I've got "Page guide at column" (set to 100 in my case), but the non-fixed-width Helvetica seems to ignore it. 🤷‍♂